### PR TITLE
feat: Adds token authentication content to the Account page

### LINKED ISSUE #181: Add token authentication content to Account page
Update Account page to match [this comp](https://www.figma.com/file/LZVQecGplb4Lnuh7a24Jef/GTFS-Tool-Forms?type=design&node-id=165-37&mode=dev). 

Include a curl command snippet for how to authenticate.

### PR DESCRIPTION
**Pull request by davidgamez**
**Summary:**

Adds token authentication content to the Account page.

Closes #181 

**Expected behavior:** 

The code snippet for generating the access token should be visible on the Account page.

**Testing tips:**

- Sign in 
- On the account page, verify that access token generation in visible
- Click on copy to clipboard(access token section)
- Open a terminal and paste the command
- Verify that the API URL is pointing to the production environment, https://api.mobilitydatabase.org/v1/tokens
- If you are testing in DEV or QA, replace `api` hostname with `api-dev` or `api-qa`
- Hit enter and expect a JSON response with a valid access token.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./scripts/api-tests.sh` to make sure you didn't break anything
- [x] Add or update any needed documentation to the repo
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [x] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

**Comment by davidgamez** [comment]
Screenshot 2024-01-03 at 11 56 27 AM

**Comment by cka-y** [comment]
shouldn't the of the component be `How to Get the Refresh Token in Your App`?

File: web-app/src/app/screens/Account.tsx [hunk 1]
```diff
@@ -589,17 +592,91 @@ export default function APIAccount(): React.ReactElement {
...
               &apos;Authorization: Bearer [Your Access Token]&apos;
             </Typography>
           </Paper>
+          <Box sx={{ p: 1, mt: 5 }}>
+            <Typography variant='sectionTitle'>
+              How to Get the Access Token in Your App
+            </Typography>
+            <Typography sx={{ mb: 2 }}>
+              Follow{' '}
+              <Link href='mailto:api@mobilitydata.org' color={'inherit'}>
```
Comment (cka-y) [inline web-app/src/app/screens/Account.tsx:616]: this link should be pointing to the swagger documentation and open in a new page

**Comment by davidgamez** [comment]
> shouldn't the title of the component be `How to Get the Refresh Token in Your App`?

The _Refresh_ token is per user and needs to be provided. The process documented is actually to programmatically obtain the access token without accessing the UI. For this, the request for the refresh token is needed.

**Review by cka-y — COMMENTED** [review]
I get error `Invalid GCIP ID token: empty token` -- is this expected since we are testing prod endpoints from dev?

**Review by cka-y — APPROVED** [review]
LGTM!

**Pull request merged**